dpgen/generator/run.py

```diff
@@ -460,7 +460,7 @@ def run_train (iter_index,
     except KeyError:
         mdata = set_version(mdata)
 
-    
+
```

Comment:
Please do not introduce such space removing revision. Thanks!

Reply:
My vim editor automatically removes the extra spaces, any suggestion in this scenario?

Reply:
all removed